### ISSUE TITLE
Ads being classified as trackers

### ISSUE THREAD

**Issue by RuthlessRuler** [brave/browser-android-tabs#749]
**Did you search for similar issues before submitting this one?**
Yes

**Description:**
Though the ads are being blocked but on the new tab the ad counts persists to zero and all the ads+ trackers count is incremented in Trackers Category.

**Device Details:**
   - Install Type - ARM:
   - Device - Phone - Galaxy Core Duos:
   - Android Version - Jelly Bean: 
    
**Brave Version:**
1.0.58

**Steps to reproduce:**
    1. Visit A website which have both trackers and Ads
    2. Check the count of Ads And Trackers (Suppose y) The Brave Shield
    3. Open A New Tab and The Count of ads = 0 , Trackers = Earlier + y

**Actual Behavior**
Ads not being classified properly, so it remains at 0.

**Expected Behavior**
Ads are classified properly and therefore the counter increases as the ads are blocked

**Is this an issue with Beta build?**
Nope

**Is this an issue in the currently released version?**
Yes

**Can this issue be consistently reproduced?**
Cant Say as it doesnt reproduce on other Android Device i Own.

**Screenshot:**
![screenshot_2018-09-19-16-39-09](https://user-images.githubusercontent.com/20885012/45749913-cd12a680-bc2a-11e8-9b65-417acb879942.png)

**Comment by SergeyZhukovsky** [comment]
Do you have 0 ads always? Could you provide an example of the website?

**Comment by RuthlessRuler** [comment]
Yes! Its always 0 on the new tab page! Eg. Youtube or any site that has sort of any Ads!

**Comment by SergeyZhukovsky** [comment]
Do you have the global check box turned off? Check please Settings->Privacy->Ad Block.

**Comment by RuthlessRuler** [comment]
No! Its turned on, 
Plus the ads are getting blocked!
